include/py_functions.py

```diff
@@ -1,22 +1,7 @@
-import ConfigParser
```

Comment:
Please extract these changes to a separate PR (it's an independent change necessary for Python3 to work).